tests/subsys/debug/coredump_backends/testcase.yaml

```diff
@@ -12,13 +12,6 @@ tests:
     platform_exclude: acrn_ehl_crb
     extra_configs:
       - CONFIG_TEST_STORED_COREDUMP=n
-    harness: console
-    harness_config:
-      type: multi_line
-      regex:
-        - "E: #CD:BEGIN#"
-        - "E: #CD:5([aA])45([0-9a-fA-F]+)"
-        - "E: #CD:END#"
```

Comment:
If you need to remove this, please find an alternative to test the output from console. Without this, we don't know if the basic coredump is working correctly.

Reply:
we already have another test suite for the logging backend's output (https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/subsys/debug/coredump/testcase.yaml)

So, in this `coredump_backend` suite, we can focus on backend commands/queries and for the logging backend it is only 'get error' by  `test_coredump_0_ready` right after the crash.

Reply:
Ok. That sounds reasonable.